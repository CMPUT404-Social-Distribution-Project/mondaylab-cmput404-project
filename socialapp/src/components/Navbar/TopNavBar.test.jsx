import React from "react";
import { render } from '@testing-library/react';
import TopNavBar from './TopNavbar';
import { BrowserRouter } from "react-router-dom";

describe('Testing Top NavBar:', () => {
    it("renders correctly", () => {
        const tree = render(<BrowserRouter><TopNavBar /></BrowserRouter>);
        expect(tree).toMatchSnapshot();
    });
});