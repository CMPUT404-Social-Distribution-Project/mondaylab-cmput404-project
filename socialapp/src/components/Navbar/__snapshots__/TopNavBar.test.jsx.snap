// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Top NavBar: renders correctly 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <nav
        class="navbar navbar-expand navbar-dark bg-teal-alt"
        id="topnav"
      >
        <div
          class=".me-1 container-fluid"
        >
          <span
            class="me-auto navbar-brand"
          >
            <img
              alt="logo"
              src="logo.svg"
              style="height: 50px; width: 50px;"
            />
            <img
              alt="logo-text"
              src="logo-text.svg"
              style="height: 50px; width: 250px;"
            />
          </span>
          <div
            class="ml-auto navbar-nav"
          >
            <span
              id="profilePicContainer"
            >
              <img
                alt="profilepic"
                id="profilePic"
              />
            </span>
          </div>
        </div>
      </nav>
    </div>
  </body>,
  "container": <div>
    <nav
      class="navbar navbar-expand navbar-dark bg-teal-alt"
      id="topnav"
    >
      <div
        class=".me-1 container-fluid"
      >
        <span
          class="me-auto navbar-brand"
        >
          <img
            alt="logo"
            src="logo.svg"
            style="height: 50px; width: 50px;"
          />
          <img
            alt="logo-text"
            src="logo-text.svg"
            style="height: 50px; width: 250px;"
          />
        </span>
        <div
          class="ml-auto navbar-nav"
        >
          <span
            id="profilePicContainer"
          >
            <img
              alt="profilepic"
              id="profilePic"
            />
          </span>
        </div>
      </div>
    </nav>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
